src/components/counselor-table.tsx

```diff
@@ -2,135 +2,58 @@ import React, { RefObject } from "react";
-  "First-Name": "First",
-  "Middle-Name": "Middle",
-  "Name-Suffix": "Suffix",
-  "Business-Name": "Business",
-  "License-Active-Status-Description": "Active?",
-  County: "County Num",
-  "County-Description": "County Name",
-  "Mailing-Address-Line1": "Mailing Address",
-  "Mailing-Address-line2": "Mailing Address line 2",
-  "Mailing-Address-City": " Mailing City",
-  "Mailing-Address-State": "Mailing State",
-  "Mailing-Address-ZIPcode": "Mailing ZIP",
-  "Mailing-Address-Area-Code": "Mailing Area-code",
-  "Mailing-Address-Phone-Number": "Mailing Phone Num",
-  "Mailing-Address-Phone-Extension": "Mailing Ext.",
-  "Practice-Location-Address-Line1": "Practice Address",
-  "Practice-Location-Address-line2": "Practice Address Line 2",
-  "Practice-Location-Address-City": "Practice City",
-  "Practice-Location-Address-State": "Practice State",
-  "Practice-Location-Address-ZIPcode": "Practice ZIP",
-  Email: "Email",
-  "Mod-Cdes": "Mod Codes",
-  "Prescribe-Ind": "Prescribe?",
-  "Dispensing-Ind": "Dispensing?"
-};
-
-const WIDTH_MAPPING = {
-  lic_id: 80,
-  "Expire-Date": 120,
-  "Original-Date": 120,
-  "License-Number": 80,
-  "Status-Effective-Date": 120,
-  "Board-Action-Indicator": 150,
-  "License-Status-Description": 140,
-  "Last-Name": 140,
-  "First-Name": 140,
-  "Middle-Name": 80,
-  "Name-Suffix": 60,
-  "Business-Name": 80,
-  "License-Active-Status-Description": 80,
-  County: 40,
-  "County-Description": 140,
-  "Mailing-Address-Line1": 300,
-  "Mailing-Address-line2": 300,
-  "Mailing-Address-City": 200,
-  "Mailing-Address-State": 130,
-  "Mailing-Address-ZIPcode": 110,
-  "Mailing-Address-Area-Code": 180,
-  "Mailing-Address-Phone-Number": 180,
-  "Mailing-Address-Phone-Extension": 120,
-  "Practice-Location-Address-Line1": 300,
-  "Practice-Location-Address-line2": 300,
-  "Practice-Location-Address-City": 200,
-  "Practice-Location-Address-State": 130,
-  "Practice-Location-Address-ZIPcode": 110,
-  Email: 240,
-  "Mod-Cdes": 200,
-  "Prescribe-Ind": 110,
-  "Dispensing-Ind": 120
-};
-
-const TABLE_WIDTH = Object.keys(WIDTH_MAPPING).reduce(
-  (total, key) => total + WIDTH_MAPPING[key],
-  0
-);
-
 export default class CounselorTable extends React.Component<
   CounselorTableProps,
   CounselorTableState
-> {
+  > {
   props: CounselorTableProps;
   state: CounselorTableState = { filteredData: this.props.data };
   wrapper: RefObject<HTMLDivElement> = React.createRef();
 
   rowGetter = ({ index }) => this.state.filteredData[index];
 
-  cellRenderer = ({ columnIndex, isScrolling, rowData }) => (
-    <div style={{ whiteSpace: "normal" }}>{rowData[columnIndex]}</div>
-  );
+  cellRenderer = ({ isScrolling, rowData, dataKey }) => {
+    return <div style={{ whiteSpace: "normal" }}>{rowData[dataKey]}</div>;
+  };
 
   updateData = list => {
-    this.setState({ filteredData: list || this.props.data });
+    this.setState({ filteredData: list ? list.map(entry => new Practitioner(entry)) : this.props.data });
```

Comment:
Will list be `undefined` or `null` here? If `undefined`, this would be cleaner by setting the param default to `[]`. If null, please ignore this comment!

Reply:
I realized you probably didn't receive a notification for this since I edited it. Please see my edits to the suggestion!

Reply:
I realized you probably didn't receive a notification for this since I edited it. Please see my edits to the suggestion!

Reply:
I realized you probably didn't receive a notification for this since I edited it. Please see my edits to the suggestion!

Reply:
Yeah I saw it. GitHub was behaving funky yesterday and my commit was not showing up. At any rate, I addressed it, I think it is ready for a final review.